docs/lib/xr/fragments/getting-started.md

```diff
@@ -21,7 +21,7 @@ Now click the *Download JSON configuration* button to download the scene configu
 
 ## Setup backend
 
-> Ensure you have [installed and configured the Amplify CLI and library](../../start/getting-started/installation.md)
+> Ensure you have [installed and configured the Amplify CLI and library](~/start/start.md)
```

Comment:
Should this be ~/start/getting-started/installation.md ?

Reply:
no, we want to redirect them to the getting-started home page, so they'll be able to choose their platform. The link works as is.